packages/commands/src/index.ts

```diff
@@ -532,6 +527,26 @@ export class CommandRegistry {
       return;
     }
 
+    // Check that only mod key(s) have been pressed.
+    if (CommandRegistry.isModifierKeyPressed(event)) {
+      // Find the exact match for the modifier keys.
+      let { exact } = Private.matchKeyBinding(
+        this._keyBindings,
+        [keystroke],
+        event
+      );
+      if (exact) {
+        // If the mod keys match an exact shortcut, start a dedicated timer.
+        event.preventDefault();
+        event.stopPropagation();
+        this._startModifierTimer(exact);
+      } else {
+        // Otherwise stop potential existing timer.
+        this._clearModifierTimer();
```

Comment:
Thanks for the ping @brichet!

Should the modifier timer also be stopped if an exact match for a chord is found?

For example if the sequence is `Ctrl Z` (not `Ctrl + Z` but `Ctrl` followed by `Z`) we would get two events (one for `Ctrl`, one for `Z`). My impression from just reading the code (I could be wrong) is that currently it would trigger both the handler for `Ctrl` and for the `Ctrl` followed by `Z` chord.

In other words, should `_clearModifierTimer` be called in `_clearPendingState` too?

Reply:
You are right, but this behavior seems OK if this feature is only used to display overlays.
You may want to keep the modifier key(s) pressed (and the overlays visible) to trigger another shortcut.

If we keep it that way, we may want to document it somewhere to avoid using this modifier key for unexpected action.

Reply:
> You may want to keep the modifier key(s) pressed (and the overlays visible) to trigger another shortcut.

Yes, but I also do not want to have the overlay visible ("sticky") after I already pressed the combo I was interested in. I think if user is pressing two shortcuts and they need overlays, say Ctrl + C and Ctrl + V it is fine it they do that in sequence:
- Ctrl: shows overlay
- Ctrl + C: overlay hides immediately
- Ctrl: shows overlay again
- Ctrl + V: overlay hides immediately again

Reply:
Actually, the overlay visible or not depends on the command that will be triggered.
Here it is only about the timeout before triggering the command, so you're right we better clear it an other command is triggered. BTW the timeout is also cleared on key up event with this PR.
I updated it.